Sources/TokamakCore/Modifiers/TextFieldStyle.swift

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Jed Fox on 06/30/2020.
+//
+
+public protocol TextFieldStyle {
```

Comment:
Since this protocol is public in SwiftUI, I think its important to match:

```swift
public protocol TextFieldStyle {
  associatedtype _Body : SwiftUI.View
  func _body(configuration: SwiftUI.TextField<Self._Label>) -> Self._Body
  typealias _Label = SwiftUI._TextFieldStyleLabel
}
```

But we could handle this in a later PR if you'd prefer...